src/cmd/run.js

```diff
@@ -323,7 +328,9 @@ export class ExtensionRunner {
   }
 
   run(profile: FirefoxProfile): Promise<FirefoxProcess> {
-    const {firefoxApp, firefox} = this;
-    return firefoxApp.run(profile, {firefoxBinary: firefox});
+    const {firefoxApp, firefox, url} = this;
+    const binaryArgsArray = ['--url', url];
```

Comment:
it would be better to call it `binaryArgs` and we should add the `url` to the `binaryArgs` only if it is defined.

Also we should keep into account that #706 is going to introduce its own elements in the `binaryArgs` array and we should do it using the same strategy, and as a nice side-effect it will make easier merging/rebasing these two pull request regardless of the landing order, here is where the similar change is applied in #706 https://github.com/aniketkudale/web-ext/blob/4f17f3d11d6602aa31ab00010f58d91284971d5d/src/cmd/run.js#L329-L338)